src/azure-cli/requirements.py3.windows.txt

```diff
@@ -117,6 +117,7 @@ oauthlib==3.0.1
 packaging==21.3
 paramiko==2.10.1
 pbr==5.3.1
+pkginfo==1.8.2
```

Comment:
`pkginfo` is actually already installed via:

https://github.com/Azure/azure-cli/blob/b4881c2d96a477450cf240379d9df0357aefa2c5/src/azure-cli/requirements.py3.windows.txt#L36

https://github.com/Azure/azure-cli/blob/329e6786d748c37347c725e9da88c824520d144b/src/azure-cli-core/setup.py#L59

but yes, it is a good idea to add it here too.